sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/ChatbotSample.java

```diff
@@ -8,26 +8,22 @@
 import com.azure.core.credential.AzureKeyCredential;
 
 /**
- * Sample demonstrating the minimal usage where, aside from providing authentication
- * details, we only need to pass a prompt and the deploymentId
+ * A sample demonstrating the minimal usage with default credentials 
+ * where we only need to pass a prompt and the deploymentId
  */
-public class GetCompletionsFromPrompt {
-
+public class ChatbotSample {
     /**
-     * The sample will return the text choices that are generated based on the prompt provided by the user
-     *
      * @param args Unused. Arguments to the program.
      */
     public static void main(String[] args) {
-        String azureOpenAIKey = "{azure-open-ai-key}";
         String endpoint = "{azure-open-ai-endpoint}";
-        String deploymentOrModelId = "{azure-open-ai-deployment-model-id}";
-
+        TokenCredential defaultCredential = new DefaultAzureCredentialBuilder().build();
```

Comment:
DefaultAzureCredentialBuilder will build a AAD TokenCredential. For sample, it is better to use an easy-to-get authentication, we should use         

String azureOpenAIKey = "{azure-open-ai-key}";